server/src/internalClusterTest/java/org/opensearch/search/aggregations/bucket/TermsShardMinDocCountIT.java

```diff
@@ -88,6 +88,10 @@ private static String randomExecutionHint() {
 
     // see https://github.com/elastic/elasticsearch/issues/5998
     public void testShardMinDocCountSignificantTermsTest() throws Exception {
+        assumeFalse(
```

Comment:
this test is validating the count of buckets returned with size and shardSize parameter. I would think that bucket count should be same in both the cases ?

Reply:
Significant terms aggregations return the top `size` buckets by significance score. The `min_doc_count` and it's shard counterpart are therefore used to calibrate the "certainty" that a significant term is in fact significant. For example, using a higher `shard_min_doc_count` could help filter out typos in the term field.

So for this significant terms test if we increase the `shard_size` then the shards will return more buckets and some buckets may have doc count greater than `min_doc_count`, so they will get returned by the coordinator and fail the first set of assertions.